cli/waiter/version.py

```diff
@@ -0,0 +1 @@
+VERSION = '0.0.1'
```

Comment:
Are we going to have separate snapshot and release versions?

Reply:
We can if you think there is a benefit to it.

Reply:
I prefer having snapshots and then an actual release version. Otherwise there is a burden on every commit making a change to the cli of updating the version.

Reply:
Done